web/client/examples/rasterstyler/assets/css/mobile.css

```diff
@@ -0,0 +1,259 @@
+
```

Comment:
I think all these css are not needed, just include one css with the needed stuff